tests/plugins/org.polarsys.capella.test.diagram.filters.ju/src/org/polarsys/capella/test/diagram/filters/ju/cdb/HideDiagramTitleBocksTest.java

```diff
@@ -0,0 +1,43 @@
+/*******************************************************************************
+ * Copyright (c) 2020 THALES GLOBAL SERVICES.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *  
+ * Contributors:
+ *    Thales - initial API and implementation
+ *******************************************************************************/
+package org.polarsys.capella.test.diagram.filters.ju.cdb;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.polarsys.capella.core.sirius.analysis.constants.IFilterNameConstants;
+import org.polarsys.capella.test.diagram.filters.ju.DiagramTitleBlockFilterTestCase;
+
+public class HideDiagramTitleBocksTest extends DiagramTitleBlockFilterTestCase {
+
+  private final String DIAGRAM_TITLE_BLOCK_ID = "_8t01EIXwEeqWaJq2mNMsRw";
+  
+
+  @Override
+  protected String getTestProjectName() {
+    return "TitleBlocksModel";

```

Comment:
constant

Reply:
I can move this in super class. Not necessary to add constants all the time, If I know that I use the same constant in  multiple places is ok to add constants otherwise, for a single occurence is ok to directely use like that.

Reply:
ok, ignore all comments with constant in this case